### DISCUSSION TITLE
Is it possible to have buildbot report builds on master branch to github?

### DISCUSSION THREAD

**Discussion by levitte** [buildbot/buildbot#6498] [Q&A] [answered]
PR status checks is one thing, and it's clear how to set those up.

However, I also want to run regular builds of the master branch (or whatever given branch) and have such status reports sent to github as well, so that it ends up in a project tab of some sort (the same way Github Actions have a tab for this repo).

Is that possible at all with current buildbot reporters, or will I have to write my own?

**Comment by ulrich-teichert-kk** [comment, marked as answer]
I'd say look at the GitHubStatusPush reporter: http://docs.buildbot.net/current/manual/configuration/reporters/github_status.html#githubstatuspush
